documentation/src/docs/beginner_dev_guide.md

```diff
@@ -107,6 +108,11 @@ $ pravega-<version>/bin/pravega-standalone
 </p>
 </details>  
 
+**Note:** In order to remotely troubleshoot Java application, JDWP port is being used. The default JDWP port in Pravega(8050) can be overwritten using the below command before starting Pravega in standalone mode.
```

Comment:
nit: This perhaps should be "overridden" and not "overwritten"

Reply:
Thanks Prajakta, updated document accordingly.